kin-backup-and-restore/kin-backup-and-restore-lib/src/main/res/values/strings.xml

```diff
@@ -48,6 +48,8 @@ To do this, you’ll first need to create a password and save your unique QR cod
     <string name="backup_and_restore_save_your_qr_code_description">Now, save your unique QR code by emailing it to yourself.
 You\'ll need it to load your Kin from a backup later on.</string>
     <string name="backup_and_restore_my_kin_wallet_qr_code">Save your QR code</string>
+    <string name="your_app_qr_code_subject">Your %1$s QR code</string>
+    <string name="your_app_qr_code_body">Your %1$s QR code is attached. When you restore your account on %1$s, you will need to provide this QR code. Please save this QR code.</string>
```

Comment:
Who gave you those string?
Is it your decision or someone else?

Reply:
its liat requests